cmd/main.go

```diff
@@ -94,8 +105,29 @@ func (f *sharedFlags) addFlags(cmd *cobra.Command) error {
 	return nil
 }
 
+type sharedJWTFlags struct {
+	audience        string
+	endpoint        string
+	sessionDuration int
+	workloadAPIAddr string
+}
+
+func (f *sharedJWTFlags) addFlags(cmd *cobra.Command) error {
+	cmd.Flags().StringVar(&f.audience, "audience", "", "Sets what audience will be used for the JWT. Required.")
+	if err := cmd.MarkFlagRequired("audience"); err != nil {
+		return fmt.Errorf("marking audience flag as required: %w", err)
+	}
+	cmd.Flags().StringVar(&f.endpoint, "endpoint", "", "The URL of the IAM endpoint. Required.")
```

Comment:
Maybe it'd be nice to have this default to the value for standard AWS since that's the primary use-case for this tool. But, I'm happy enough to handle that myself in a separate PR.